lib/scenic/adapters/postgres.rb

```diff
@@ -155,17 +155,36 @@ def create_materialized_view(name, sql_definition, no_data: false)
       # @param no_data [Boolean] Default: false. Set to true to create
       #   materialized view without running the associated query. You will need
       #   to perform a non-concurrent refresh to populate with data.
+      # @param side_by_side [Boolean] Default: false. Set to true to create the
+      #   new version under a different name and atomically swap them, limiting
+      #   downtime at the cost of doubling disk usage
       #
       # @raise [MaterializedViewsNotSupportedError] if the version of Postgres
       #   in use does not support materialized views.
       #
       # @return [void]
-      def update_materialized_view(name, sql_definition, no_data: false)
+      def update_materialized_view(
+        name, sql_definition, no_data: false, side_by_side: false
```

Comment:
I'm not sure `side_by_side` is the name I'd opt for here but I'm lacking a better suggetion at the moment other than `background` which I don't love either.

In your experience, is `side_by_side` an established name for this process?

Reply:
I think "online" might be a more established name but I was trying to think of something that was very specific about what it did